Examples/Python/python/acts/examples/simulation.py

```diff
@@ -493,45 +483,38 @@ def addFatras(
     # Sequencer
     s.addAlgorithm(alg)
 
-    # Selector
-    if postSelectParticles is not None:
-        particlesInitial = "fatras_particles_initial_selected"
-        particlesFinal = "fatras_particles_final_selected"
-        addParticleSelection(
-            s,
-            postSelectParticles,
-            inputParticles=outputParticlesInitial,
-            inputParticlesFinal=outputParticlesFinal,
-            outputParticles=particlesInitial,
-            outputParticlesFinal=particlesFinal,
-        )
-        s.addWhiteboardAlias("particles_selected", particlesInitial)
-    else:
-        particlesInitial = outputParticlesInitial
-        particlesFinal = outputParticlesFinal
-
-    # Only add alias for 'particles_initial' as this is the one we use most
-    s.addWhiteboardAlias("particles", outputParticlesInitial)
-
     # Output
     addSimWriters(
```

Comment:
Hmm this changes which particles are written, right? before only teh selection was written, now we write before the selection. Is there a reason why this is changed?

Reply:
Good point! I will revert this here as it does not need to be coupled with this change.

I was in doubt if we should write the selected or raw particles. At the same time we write all the hits so I feel like we are loosing information?

I believe the right solution is to elevate the particle selection into the user's chain and not have it in the simulation.